paxoskv/impl.go

```diff
@@ -265,6 +269,8 @@ func (s *KVServer) Accept(c context.Context, r *Proposer) (*Acceptor, error) {
 		LastBal: &d,
 	}
 
+	// article say acceptor's LastBal equal proposer's Bal will accept it
+	// but if greater, point that a large proposer's Bal has been through phrase1 with most acceptor, the same accept it
```

Comment:
Great explain! Thanks man!

Reply:
> Great explain! Thanks man!

I read your article then find this code. code is not robust when grpc make communication, but that is enough to studyappreciate！